i18n/en.json

```diff
@@ -685,7 +685,7 @@
   },
   {
     "id": "api.context.invalid_team_url.debug",
-    "translation": "TeamURL accessed when not valid. Team URL should not be used in api functions or those that are team independent"
+    "translation": "Team URL accessed when not valid. Team URL should not be used in API functions or those that are team independent"
```

Comment:
@jwilander can you help clarify? This message is a bit confusing,

Wondering if it should be:
"Team URL should not be used in API functions."
or
"Team URL should not be used in API functions that are team independent."